libgalois/include/katana/PropertyGraph.h

```diff
@@ -98,15 +117,18 @@ class KATANA_EXPORT GraphTopology {
   /// \param node node to get the edge range of
   /// \returns iterable edge range for node.
   edges_range edges(Node node) const {
-    auto [begin_edge, end_edge] = edge_range(node);
-    return MakeStandardRange<edge_iterator>(begin_edge, end_edge);
+    return MakeStandardRange<edge_iterator>(edge_begin(node), edge_end(node));
   }
 
   Node edge_dest(Edge edge_id) const {
     KATANA_LOG_DEBUG_ASSERT(edge_id < dests_.size());
     return dests_[edge_id];
   }
 
+  Node edge_dest(const edge_iterator ei) const noexcept {
```

Comment:
If this can be a `noexcept`, so can the function it calls, right?

Reply:
Yes, they all should be noexcept :)